desktop/src/main/java/bisq/desktop/main/dao/bonding/unlock/LockupTxListItem.java

```diff
@@ -88,7 +99,18 @@
     }
 
     public boolean isLockupAndUnspent() {
-        return !isSpent() && getTxType() == TxType.LOCKUP;
+        boolean isLocked;
```

Comment:
I've tried to avoid using 'locked' as a key word, it's not entirely clear. Better to use 'lockup', 'unlocking', 'unlocked' or more specific words for clarity.

Reply:
ok

Reply:
i will try to use a state enum, the booleans are not covering the states well as u need to combine a few...